src/main/flight/pid.c

```diff
@@ -418,7 +418,7 @@ static FAST_RAM_ZERO_INIT float acroTrainerGain;
 
 void pidUpdateAntiGravityThrottleFilter(float throttle)
 {
-    if (antiGravityMode) {
+    if ((antiGravityMode == ANTI_GRAVITY_SMOOTH) && antiGravityEnabled)) {
```

Comment:
One too many `)`. See also: Failing tests.

Reply:
Forgive my haste.  Should have at least ran a build before pushing.